src/@types/Commands.ts

```diff
@@ -0,0 +1,13 @@
+export enum Command {
+  bold = 'bold',
+  italic = 'italic',
+  underline = 'underline',
+  strikethrough = 'strikethrough',
+}
+
+export const commands = Object.values(Command)
+
+/** The set of commands applied to text (bold, italic, underline, strikethrough). */
+type Commands = Record<Command, boolean>
```

Comment:
Let's call this `CommandState`, as `Command` is overly general and could get mixed up with the commands in the `CommandPalette`.

Reply:
Makes good sense to me. Done: 781947f7562123830d3d8c4abdc17783a553b11d